daisy/step_create_disks.go

```diff
@@ -76,3 +84,28 @@ func (c *CreateDisks) run(ctx context.Context, s *Step) DError {
 		return nil
 	}
 }
+
+// Merges two slices of features and returns a new slice instance.
+// Duplicates are removed, and the result is sorted lexically.
+func combineGuestOSFeatures(currentFeatures []*compute.GuestOsFeature,
```

Comment:
current/additional doesn't matter here since you're creating a return value so naming can me features1, features2

Reply:
sgtm